pythermalcomfort/__init__.py

```
@@ -1 +1 @@
-__version__ = "3.2.0"
+__version__ = "3.3.0"
```

Pylint (3.3.7)

[convention] Missing module docstring

(C0114)